resources/views/users/getquestions.blade.php

```diff
@@ -9,21 +9,52 @@
                 <p>Check Your Brain</p>
             </div>
             <form method="POST" name="quizanswer" action="{{ route('user.dashboard.store') }}">
-                @foreach ($quizQuestions as $questionKey => $quizQuestion)
-                    @php $questionId = $quizQuestion->id @endphp
-                    @csrf
-                    @php $options = $quizQuestion->quiz_options @endphp
-                    <h4> Q{{$loop->iteration}}.{{ $quizQuestion->question }}</h4>
-                    <br>
-                    @foreach ($options as $key => $option)
-                        <input type="radio" name={{ "quizanswers[$questionId]" }} value="{{ $option->id }}">{{ $option->option }} <br>
-                    @endforeach
-                @endforeach
+                @csrf
+                <div class="container mb-5">
+                    <div class="row"> 
+                       <div class="col-12">
+                        @foreach ($quizQuestions as $questionKey => $quizQuestion)
+                          <p class="fw-bold">Q{{$loop->iteration}}.{{ $quizQuestion->question }}</p> 
+                             <div> 
+                                @php $questionId = $quizQuestion->id @endphp
+                                @csrf
+                                @php $options = $quizQuestion->quiz_options @endphp
```

Comment:
one @php and @endphp is enough.
kindly change it

Reply:
this changed  in next commit please check it